docs/pages/preview/moderated-sessions.mdx

```diff
@@ -0,0 +1,200 @@
+- Join SSH and Kubernetes sessions from the terminal as an observer, moderator or peer.
+- Require auditors to be present in certain sessions.
+- Allow people to join your session to observe what you are doing in real time.
+
+## Getting Started
+
+In this guide we'll be using Moderated Sessions to define required participants for a Kubernetes session.
+This guide assumes an already running Teleport cluster with a connected Kubernetes cluster named `minikube`.
+
+<Admonition type="tip">
+  If you don't have a cluster that meets the aforementioned requirements,
+  you can create one by following [this guide](https://goteleport.com/docs/kubernetes-access/getting-started/).
+</Admonition>
+
+### Step 1/4: Create the prod-access role
+
+```sh
+cat > prod-access.yaml <<EOF
+kind: role
+metadata:
+  name: prod-access
+spec:
+  allow:
+    kubernetes_groups:
+      - system:masters
+    require_session_join:
+      - name: Audtor oversight
+        filter: 'contains(user.roles, "auditor")'
+        kinds: ['k8s']
+        modes: ['moderator']
+EOF
+
+tctl create -f prod-access.yaml
+```
+
+### Step 2/4: Create the auditor role
+
+```sh
+cat > auditor.yaml <<EOF
+kind: role
+metadata:
+  name: auditor
+spec:
+  allow:
+    join_sessions:
+    - kinds:
+      - k8s
+      modes:
+      - moderator
+      name: Auditor
+      roles:
+      - member
+EOF
+
+tctl create -f auditor.yaml
+```
+
+### Step 3/4: Add users
+
+```sh
+tctl users add adam --roles=prod-access
+tctl users add bob --roles=auditor
+```
+
+### Step 4/4: Try it out
+
+Please substitute `$SESSION_ID` with the UUID shown when starting the session.
+
+```sh
+# adam
+
+tsh kube login minikube
+tsh kube exec -st nginx /bin/bash
+```
+
+```sh
+# bob
+
+tsh kube join $SESSION_ID
+```
+
+## RBAC
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and who are required to be present to start one.
+
+The system is based around **require policies** and allow policies. Require policies define a set of conditions
+that must be a met for a session to start/run.
+
+**Allow policies** are used to define what a user can do. In essence, what sessions they can join
+and under what conditions they may join the session.
+
+### Session kinds
+
+Require and allow policies have to specify which sessions they apply to. Valid options are `ssh` and `k8s`.
+
+### Filters
+
+Require policies can specify which users they consider as valid with a filter expression. They work like
+other filters used within Teleport such with access requests. The filter context has a `user` object defined
+which the set fields `roles`, `traits` and `name`.
+
+### Backwards compatibility with Server Access
+
+Previously, Server Access did not include controls over which users can join a session.
+To work around this, RBAC rules are ignored for users that only have V4 roles.
+New roles are created as V5 and V4 roles are also upgraded when they are modified in the UI.
```

Comment:
```suggestion
New roles are created as V5. V4 roles are upgraded when they are modified in the UI.
```
I originally mis-parsed this as "as (V5 and V4 roles)," so I've suggested this to make this clearer.